Gemfile

```diff
@@ -7,7 +7,7 @@ gem 'rails-controller-testing' # Included to provide 'assigns' method
 
 gem 'dynamic_form'
 gem 'passenger', '~> 4.0.41', group: :production
-gem 'rails', '= 5.0.6'
+gem 'rails', '~> 5.1.0'
```

Comment:
it would be more correct to make this `~> 5.1.6`. We are fine will moving forward to 5.1.7+ but we'd never want to move backwards to 5.1.5 and below because they presumably have bugs that were fixed in 5.1.6 and we haven't tested against them. However in practice this would never happen so its just a nit.